core/templates/pages/topic-editor-page/modal-templates/create-new-story-modal.component.html

```diff
@@ -22,55 +26,64 @@ <h3>
-             ng-class="{'is-invalid': storyUrlFragmentExists || !validUrlFragmentRegex.test(story.urlFragment)}"
-             ng-change="onStoryUrlFragmentChange()"
+      <input class="form-control protractor-test-new-story-url-fragment-field"
+             [(ngModel)]="newlyCreatedStory.urlFragment"
+             [ngModelOptions]="{standalone: true}"
+             [maxlength]="MAX_CHARS_IN_STORY_URL_FRAGMENT"
+             [ngClass]="{'invalid-field': storyUrlFragmentExists || !validUrlFragmentRegex.test(newlyCreatedStory.urlFragment)}"
+             (change)="onStoryUrlFragmentChange()"
              aria-label="Url fragment">
       <div>
         <span class="oppia-input-box-subtitle">
           <em>
-            The story URL fragment is used to uniquely access the story viewer page. It should consist of one or more hyphen-separated words, all in lowercase, with at most <[MAX_CHARS_IN_STORY_URL_FRAGMENT]> characters in total and must be unique across the topic. Please use meaningful keywords, and avoid using words like "and", "of", or "the".
+            The story URL fragment is used to uniquely access the story viewer page. It should consist of one or more hyphen-separated words, all in lowercase, with at most {{MAX_CHARS_IN_STORY_URL_FRAGMENT}} characters in total and must be unique across the topic. Please use meaningful keywords, and avoid using words like "and", "of", or "the".
             This story can be accessed at the following URL:<br>
-            <[hostname]>/learn/<[classroomUrlFragment]>/<[topicUrlFragment]>/story/<[story.urlFragment]>
+            {{hostname}}/learn/{{classroomUrlFragment}}/{{topicUrlFragment}}/story/{{newlyCreatedStory.urlFragment}}
           </em>
         </span>
-        <div ng-if="storyUrlFragmentExists" class="oppia-input-box-subtitle text-danger">
+        <div *ngIf="storyUrlFragmentExists" class="oppia-input-box-subtitle text-danger">
           <em>
             This story URL fragment already exists.
           </em>
         </div>
       </div>
     </div>
+
     <div class="story-input story-input-thumbnail">
       <div class="story-input-header">
         <span>Thumbnail Image*</span>
       </div>
-      <oppia-thumbnail-uploader [preview-title]="story.title"
-                                [aspect-ratio]="'4:3'"
+      <oppia-thumbnail-uploader [previewTitle]="newlyCreatedStory.title"
+                                [aspectRatio]="'4:3'"
                                 [disabled]="false"
-                                [use-local-storage]="true"
-                                [allowed-bg-colors]="allowedBgColors"
-                                [thumbnail-bg-color]="story.getThumbnailBgColor()"
-                                preview-description-bg-color="#2F6687"
-                                (image-save)="updateView()">
```

Comment:
You need to keep this `updateView` function -- that is probably why the thumbnail is not updating.

Reply:
+1. The video in the description is not the expected result.